### DISCUSSION TITLE
[Feature]: Specify Custom Font in Share Chatbot Panel?

### DISCUSSION THREAD

**Discussion by dkindlund** [FlowiseAI/Flowise#1515] [Ideas]
I realize this might need to be a feature for both this project and the FlowiseEmbedChat project, but I'm putting the feature idea here, because ultimately the UI to specify this configuration lives within the top-level Flowise project.

Specifically, I'd love an easy to way to swap in different fonts for the chatbot as needed.

For example, if there was an easy way to use this font as an option:
https://github.com/githubnext/monaspace

**Comment by saatchi-david** [comment]
Agree, it would be nice to be able to specify somewhere, e.g. in the embed config.
